src/renderers/webgpu/utils/WebGPUUtils.js

```diff
@@ -109,6 +109,20 @@ class WebGPUUtils {
 
 	}
 
+	getPreferredCanvasFormat() {
+
+		if ( navigator.userAgent.includes( 'Quest' ) ) {
```

Comment:
FYI @cabanier

Reply:
> FYI @cabanier

Thanks! lgtm
Is this because Quest is returning the wrong preferred format?

Reply:
Yes.

Will we be able to remove this check soon?

Reply:
> Yes.
> 
> Will we be able to remove this check soon?

I'll take a look

Reply:
> Yes.
> 
> Will we be able to remove this check soon?

The next version of browser will have a fix for this.